README.md

```diff
@@ -1,7 +1,7 @@
 # cltdevs meetup manifesto
 
 ## Mission statement
-We grow Charlotte's tech community by bringing people together.
+We are committed to the Charlotte tech community's growth and sustainability by fostering relationships between individuals and organizations, cultivating individual talents and organizations, and discover and build the missing elements of a thriving tech community.
```

Comment:
> and discover and build
Should it be "discovering and building" to stay consistent with the -ing verbs mentioned before?